src_pop/core/biogeochem/POP.F90

```diff
@@ -2005,16 +2075,25 @@ SUBROUTINE Patch_partial_disturb2(pop,idisturb)
   END SUBROUTINE Patch_partial_disturb2
 
 
-  !*******************************************************************************
-
+  !------------------------------------------------------------------------------------
 
+  !! Patch Disturbance
```

Comment:
did you really want these !! lines here? They appears as a rogue comment at the top pf the POP MODULE page - not in the Patch_disturb box

Reply:
These comments were indeed redundant and at the wrong place. I have removed them.